sunpy/time/tests/test_time.py

```diff
@@ -13,130 +14,170 @@
 
 import pytest
 
-LANDING = datetime(1966, 2, 3)
+LANDING = ap.Time('1966-02-03', format='isot')
 
 
 def test_parse_time_24():
-    assert parse_time("2010-10-10T24:00:00") == datetime(2010, 10, 11)
+    dt = parse_time("2010-10-10T24:00:00")
+    assert dt == ap.Time('2010-10-11')
+    assert dt.format == 'isot'
+    assert dt.scale == 'utc'
 
 
 def test_parse_time_24_2():
-    assert parse_time("2010-10-10T24:00:00.000000") == datetime(2010, 10, 11)
+    dt = parse_time("2010-10-10T24:00:00.000000")
```

Comment:
what about a test with a leap second? does that work as `24:00:01`? I can't remember.

Reply:
In one of the list test, there is a leap second example. I'll add an explicit one too.